pkg/errors/catcher.go

```diff
@@ -178,14 +179,14 @@ func CatchCCloudV2Error(err error, r *http.Response) error {
 
 	if resBody.Message != "" {
 		message := strings.TrimRight(resBody.Message, "\n")
-		return fmt.Errorf("%s: %w", message, err)
+		return errors.New(message)
 	}
 
 	if resBody.Error.Message != "" {
 		message := strings.TrimFunc(resBody.Error.Message, func(c rune) bool {
 			return c == '.' || c == '\n'
 		})
```

Comment:
Unrelated to the PR, but https://pkg.go.dev/strings#TrimRight would be more concise! It takes a "cutset" which we could set to ".\n"